### ISSUE TITLE
CSS & JS assets broken link if project is installed in subdirectory

### ISSUE THREAD

**Issue by oranog** [orchidsoftware/platform#2237]
**Describe the bug**
I've installed my project to sub directory of localhost. Now link to my project looks like this - http://localhost/myorchidproject/

When i try to open it, i'm getting assets errors in console that css & js can't be found. It seems that my project directory is not included in assets link - http://localhost/vendor/orchid/css/orchid.css

**To Reproduce**

  1. Installed Laravel & Orchid as stated in official docs - https://orchid.software/en/docs/installation/

**Expected behavior**
I expect links to assets to be http://localhost/myorchidproject/vendor/orchid/css/orchid.css instead of http://localhost/vendor/orchid/css/orchid.css

**Comment by tabuna** [comment]
Make sure your app's base url matches `http://localhost/myorchidproject/`

```bash
// .env

APP_URL=http://localhost/myorchidproject
```

**Comment by oranog** [comment]
> Make sure your app's base url matches `http://localhost/myorchidproject/`
> 
> ```shell
> // .env
> 
> APP_URL=http://localhost/myorchidproject
> ```

Yes, this is correct, but it does not help. Still pointing assets to root localhost directory.
